asv/www/regressions.js

```diff
@@ -113,11 +122,19 @@ $(document).ready(function() {
             });
         });
 
+        var branch_index = 0;
         if (branches && branches.length > 1) {
-            current_title = "Regressions in " + branches[0] + " branch";
+            var info = $.asv.parse_hash_string(window.location.hash);
+            if (info.params.branch) {
```

Comment:
Probably no need to re-parse the location hash.
`params` is passed to this function (see line 45, 53)